src/graphql/fragments/colony.graphql

```diff
@@ -47,6 +42,15 @@ fragment Colony on Colony {
   }
 }
 
+fragment ColonyTokensConnection on ModelColonyTokensConnection {
```

Comment:
Making this a fragment lets it be used as a type which will make it obvious if something like this happens again in the future